clang/test/SemaSYCL/reqd-sub-group-size-device.cpp

```diff
@@ -14,9 +14,18 @@ queue q;
 [[intel::reqd_sub_group_size(12)]] void bar();
 [[intel::reqd_sub_group_size(12)]] void bar() {} // OK
 
+// No diagnostic because the attributes are synonyms with identical behavior.
+[[sycl::reqd_sub_group_size(12)]] void bar(); // OK
+
 // Diagnostic is emitted because the arguments mismatch.
 [[intel::reqd_sub_group_size(12)]] void quux(); // expected-note {{previous attribute is here}}
-[[intel::reqd_sub_group_size(100)]] void quux(); // expected-warning {{attribute 'reqd_sub_group_size' is already applied with different arguments}}
+[[intel::reqd_sub_group_size(100)]] void quux(); // expected-warning {{attribute 'reqd_sub_group_size' is already applied with different arguments}} expected-note {{previous attribute is here}}
```

Comment:
Is the 'expected-note' at the end of this for the conflict with the line below?

Reply:
Correct. `The previous attribute is here` on the first declaration is from the second decl conflict, and the one on the second declaration is from the third decl conflict.